src/pages/Main/BtnSlider.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+function BtnSlider({ direction, moveSlide }) {
+  return (
+    <div>
```

Comment:
감싸기 위한 요소라면 React.Fragment 활용하시는 게 좋습니다.
```suggestion
    <React.Fragment>
```
혹은 아래처럼 할 수도 있습니다.
```suggestion
    <>
```